airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -1354,6 +1354,68 @@ def run_grant_dataset_view_access(self,
                 view_project, view_dataset, view_table, source_project, source_dataset)
             return source_dataset_resource
 
+    def create_empty_dataset(self, dataset_id="", project_id="", dataset_resource=None):
+        """
+        Create a new empty dataset:
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/datasets/insert
+
+        :param project_id: The name of the project where we want to create
+               an empty a dataset. Not need to provide, if projectId in dataset_resource.
+        :type project_id: str
+        :param dataset_id: The id of dataset. Not need to provide,
+               if datasetId in dataset_resource.
+        :type dataset_id: str
+        :param dataset_resource: Dataset resource that must be provided
```

Comment:
Isn't this parameter optional? The only Required param https://cloud.google.com/bigquery/docs/reference/rest/v2/datasets#resource is `datasetid`, which we create for the user if the set `dataset_id` parameter. The rest all are optional, `access[]` array is also optional, so the required params inside it are not really required if you don't provide a daataset_resource at all.

Reply:
@kaxil, yes it is optional, I fixed description